tests/api/agent/integration/test_agent_execution.py

```
@@ -0,0 +1,246 @@
+"""Real integration tests using actual Gemini model."""
+
+import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)